src/Nest/Mapping/Types/Specialized/Attachment/AttachmentAttribute.cs

```diff
@@ -1,5 +1,8 @@
-﻿namespace Nest
+﻿using System;
+
+namespace Nest
 {
+	[Obsolete("Removed in 6.0.")]
 	public class AttachmentAttribute : ElasticsearchDocValuesPropertyAttributeBase, IAttachmentProperty
```

Comment:
I think `AttachmentAttribute` can be removed altogether in 6.x - it's used for the mapper-attachment plugin in 5.x, which is superseded by the ingest-attachment plugin. 

The `Attachment` type should remain as it's still useful for the ingest-attachment plugin.